requirements/tox-requirements.txt

```diff
@@ -0,0 +1,205 @@
+#
```

Comment:
I used pip-tools to fix the versions of dependencies. This file is the output of 
```
pip-compile requirements/tox.txt -o requirements/tox-requirements.txt --resolver=backtracking
```